ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/router-bgp-path-selection.yaml

```diff
@@ -0,0 +1,92 @@
+### Routing - BGP ###
+# Testing for address-family path-selection
+router_bgp:
+  as: 65101
+  router_id: 192.168.255.42
+  peer_groups:
+    - name: PATH-SELECTION-PG-1
+      type: ipv4
+      remote_as: 65001
+    - name: PATH-SELECTION-PG-2
+      type: ipv4
+      remote_as: 65001
+    - name: PATH-SELECTION-PG-3
+      type: ipv4
+      remote_as: 65001
+    - name: PATH-SELECTION-PG-4
+      type: ipv4
+      remote_as: 65001
+    - name: PATH-SELECTION-PG-5
+      type: ipv4
+      remote_as: 65001
+  neighbors:
+    - ip_address: 172.31.255.0
+    - ip_address: 192.168.255.1
+    - ip_address: 172.31.255.2
+    - ip_address: 172.31.255.3
+    - ip_address: 172.31.255.4
+  address_family_path_selection:
+    bgp:
+      additional_paths:
+        receive: true
+        send:
+          # Need to choose what to test here or create multiple hosts...
+          #any: true
+          #backup: true
+          #ecmp: true
+          ecmp_limit: 42
+          #limit: 42
```

Comment:
clean up comments.

Reply:
These comments are here to explain what we are not testing.